awslambdaric/bootstrap.py

```diff
@@ -370,6 +416,27 @@ def create_log_sink():
 _GLOBAL_AWS_REQUEST_ID = None
 
 
+def setup_logging(log_format, log_level, log_sink):
+    logging.Formatter.converter = time.gmtime
+    logger = logging.getLogger()
+    logger_handler = LambdaLoggerHandler(log_sink)
+    if log_format == LogFormat.JSON:
+        logger_handler.setFormatter(JsonFormatter())
+    else:
+        logger_handler.setFormatter(
+            logging.Formatter(
+                "[%(levelname)s]\t%(asctime)s.%(msecs)03dZ\t%(aws_request_id)s\t%(message)s\n",
+                "%Y-%m-%dT%H:%M:%S",
+            )
+        )
+
+    if log_level:
+        logger.setLevel(log_level)
```

Comment:
1. Should we put this part under `if log_format == LogFormat.JSON:` condition?
2. In case of incorrect value, should we gracefully handle it?

Example:
Code looks like
```
logging.addLevelName(33, "PEW")
def lambda_handler():
  logging.addLevelName(42, "PEWPEW")
```
Natural desire would be to use env variable to limit log level to `PEWPEW` or `PEW`, but at the moment of running `setup_logging` it's not yet registered.
Adding levels in a handler looks quite strange, but natural at INIT phase. If the intention is to support custom log levels, need to `setLevel` at least after INIT